packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -356,8 +358,10 @@ class HttpTextResponseViewer extends StatelessWidget {
 
         return switch (currentLocalResponseType) {
           NetworkResponseViewType.json => JsonViewer(encodedJson: responseBody),
-          NetworkResponseViewType.text => Text(
-              responseBody,
+          // Using TextViewer fixes
+          // https://github.com/flutter/devtools/issues/5973
```

Comment:
The above issue should be closed when this PR lands, but I'd file a new issue for this as it's a bandaid fix if we don't know why we're seeing the hang in the first place.

Reply:
Created the following issues in response to this discussion:
- https://github.com/flutter/devtools/issues/6261
- https://github.com/flutter/devtools/issues/6263